src/pages/clients/common/components/MergeClientModal.tsx

```diff
@@ -111,7 +111,6 @@ export function MergeClientModal(props: Props) {
         onClearButtonClick={() => setMergeIntoClientId('')}
         withoutAction
         exclude={[props.mergeFromClientId]}
-        staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Merge Client Modal is no longer necessary because we have already changed it to be set by default.